theano/scalar/basic.py

```diff
@@ -3659,7 +3659,7 @@ def __init__(self, inputs, outputs):
         self.nin = len(inputs)
         self.nout = len(outputs)
         self.init_fgraph()       # self.fgraph
-
+        self.init_c_code()
```

Comment:
why did you change this file? I don't see the need and this is a bad idea. I moved this to later in the code pipeline. revert.